src/core/server/graph/schema/schema.graphql

```diff
@@ -6452,17 +6542,91 @@ input UpdateUserRoleInput {
 }
 
 type UpdateUserRolePayload {
+  """
+  clientMutationId is required for Relay support.
+  """
+  clientMutationId: String!
+
+  """
+  user is the possibly modified User.
+  """
+  user: User!
+}
```

Comment:
`UserRolePayload` wasn't actually changed, this is just a quirk in the way the diff rendered